components/esp_websocket_client/examples/target/main/websocket_example.c

```diff
@@ -146,6 +146,34 @@ static void websocket_app_start(void)
     ESP_LOGI(TAG, "Connecting to %s...", websocket_cfg.uri);
 
     esp_websocket_client_handle_t client = esp_websocket_client_init(&websocket_cfg);
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
+    esp_websocket_client_append_header(client, "aaa", "bbb");
```

Comment:
Is this intentional, or did you just want to test something?

Reply:
Thanks for catching that; it was unintentional :D